### PR TITLE
adapt plugin registry to use rhel9-2-els image

### PR DESCRIPTION
**Pull request by svor**
Use rhel9-2-els image for the build in Brew
We still have registry.redhat.io/rhel8/postgresql-15 image in the main Dockerfile for the custom plugin registry build, but for Brew it'll be replaced with registry.redhat.io/rhel9-2-els/rhel:9.2-1222 by sync.sh script 

Related issue: https://issues.redhat.com/browse/CRW-6350

### CODE REVIEW

File: devspaces-pluginregistry/build/dockerfiles/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,9 @@
 
 # https://registry.access.redhat.com/rhel8/postgresql-15
 FROM registry.redhat.io/rhel8/postgresql-15:1-66.1717586531
+# rhel9-2-els is used for the build in Brew
+# FROM registry.redhat.io/rhel9-2-els/rhel:9.2-1222
```
Comment (RomanNikitenko) [inline devspaces-pluginregistry/build/dockerfiles/Dockerfile:17]: should this instruction replace the previous one?
or
both are required?

Comment (svor) [reply devspaces-pluginregistry/build/dockerfiles/Dockerfile:17]: from the description:
```
We still have registry.redhat.io/rhel8/postgresql-15 image in the main Dockerfile for the custom plugin registry build, but for Brew it'll be replaced with registry.redhat.io/rhel9-2-els/rhel:9.2-1222 by sync.sh script
```

**Comment by SDawley** [comment]
@svor Just to confirm, has this been tested in brew?

**Comment by svor** [comment]
> @svor Just to confirm, has this been tested in brew?

@SDawley I've used this branch https://pkgs.devel.redhat.com/cgit/containers/devspaces-pluginregistry/log/?h=private-vsvydenk-rhel-els to test it in brew

**Comment by SDawley** [comment]
OK awesome thanks!

But to add to your discussion with Roman, the comment above the FROM line is used by the update script which keeps the base images up to date, so you'll either have to maintain 2 versions of the Dockerfile or update the sync script to check for newer rhel-els images before changing the FROM image.

**Comment by SDawley** [comment]
The actual script is:https://github.com/redhat-developer/devspaces/blob/devspaces-3-rhel-8/product/updateBaseImages.sh if you want to test it

**Comment by svor** [comment]
@SDawley could you please have another look at the PR:

I've created a separate [rhel.Dockerfile](https://github.com/redhat-developer/devspaces/pull/1080/files#diff-035e6683eecc9984b50cc5dd5ff997a48966aff3ffa9e9e51beadeee7fb14504).

This rhel.Dockerfile should be synced into devspace-images after merging https://github.com/redhat-developer/devspaces/pull/1080

**Comment by SDawley** [comment]
Made a similar comment on the devspaces PR, but other than needing the comment line looks good.

**Review by SDawley — APPROVED** [review]

**Pull request merged**